public/js/cookies.js

```diff
@@ -1,9 +1,29 @@
-/* global createCookie, readCookie */
+function createCookie(name, value, days) {
```

Comment:
These two (createCookie, readCookie) are declared in master-theme in global namespace